src/main/java/uk/gov/hmcts/cmc/claimstore/utils/Optionals.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class Optionals {
```

Comment:
Please remove this class, it's redundant.

Reply:
actioned